### DISCUSSION TITLE
Fan Control/Corsair Plugin won't see H80iv2

### DISCUSSION THREAD

**Discussion by Shawzborne** [Rem0o/FanControl.Releases#1985] [Q&A]
Hi Everyone,

I am trying to get the Corsair Plugin working with the Corsair H80iv2. I am running Windows 10 Pro 64 bit. I copied the two DLL files to the fan control "Plugins" directory and I also right clicked on the .inf file and hit install. But still FanControl won't see the mini usb connected H80iv2. Any suggestions or help would be appreciated.

[https://imgur.com/a/nVYfRA1](url)

Thank-you,

Shawn
![6 - 9rWfT5n](https://github.com/Rem0o/FanControl.Releases/assets/5931778/f230ba78-1e44-46a3-91b4-d94907f1ff46)
![1 - iJg4N7S](https://github.com/Rem0o/FanControl.Releases/assets/5931778/cd866f22-f215-4d67-bfe1-9203bc6690fe)
![2 - 8EUhYNx](https://github.com/Rem0o/FanControl.Releases/assets/5931778/5f64d2e2-ccf3-448d-bd94-041941ece495)
![3 - Zja5RVE](https://github.com/Rem0o/FanControl.Releases/assets/5931778/eea6fee1-901f-47c1-a36f-c797b5318e91)
![4 - HoocJ5L](https://github.com/Rem0o/FanControl.Releases/assets/5931778/5b50893e-1c06-4739-a0ea-bc9278977b85)
![5 - 2wUvu2i](https://github.com/Rem0o/FanControl.Releases/assets/5931778/adbe66e8-91fb-42ae-af74-aa2626d9ddac)

**Comment by Rem0o** [comment]
I would recommend you open this discussion on the plugin's repo:
https://github.com/EvanMulawski/FanControl.CorsairLink

The guy over usually responds quickly and addresses issues/questions swiftly.
